pkg/serverless/trace/trace.go

```diff
@@ -11,19 +11,20 @@ import (
 	"os"
 	"strings"
 
+	"github.com/DataDog/datadog-go/v5/statsd"
+
```

Comment:
nit, this probably didn't need to move

Reply:
agreed, but I have the linters enabled for pre-commit hooks so it was "easier" to let this change happen. Happy to change it back if you prefer.

Reply:
No worries, thanks for the explanation!